packages/graphql/.gitignore

```diff
@@ -0,0 +1,9 @@
+# MacOS
```

Comment:
coudl probably just use the top level gitignore, i think those are all already excluded